swath_projector/earthdata_varinfo_config.json

```diff
@@ -21,6 +23,20 @@
         }
       ],
       "_Description": "VNP10 SnowData variables have incorrect relative paths for coordinates."
+    },
+    {
+      "Applicability": {
+        "Mission": "TEMPO",
+        "ShortNamePath": "TEMPO_O3TOT_L2",
+        "VariablePattern": "/product/.*|/support_data/.*"
```

Comment:
I'm finding that utilizing the `ExcludedScienceVariables` parameter will achieve the goal of not trying to reproject the variables with dimensions not matching those of `/geolocation/latitude` and `/geolocation/longitude`.

```
  "ExcludedScienceVariables": [
    {
      "Applicability": {
        "Mission": "TEMPO",
        "ShortNamePath": "TEMPO_O3TOT_L2"
      },
      "VariablePattern": [
        "/support_data/a_priori_layer_o3",
        "/support_data/cal_adjustment",
        "/support_data/dNdR",
        "/support_data/layer_efficiency",
        "/support_data/lut_wavelength",
        "/support_data/N_value",
        "/support_data/N_value_residual",
        "/support_data/ozone_sensitivity_ratio",
        "/support_data/step_1_N_value_residual",
        "/support_data/step_2_N_value_residual",
        "/support_data/temp_sensitivity_ratio"
      ]
    }
  ],
```

This will allow me to add back in the `MissingReprojectedDataError`

Reply:
Implemented changes I described above in commit [25cdc10](https://github.com/nasa/harmony-swath-projector/pull/19/commits/25cdc1098508cbccc841bb08293697d8cc0530b4).

In this change, I didn't make the `VariablePattern` for the coordinates `MetadataOverride` more specific as you suggested, because the new `ExcludedScienceVariables` will parse out the `support_data` variables with non-matching dimensions. If you'd like to see this `VariablePattern` more specific still, I can make that change.

Reply:
I think this is a good solution. Nice work!